src/app/index.tsx

```diff
@@ -49,6 +50,7 @@ export function App() {
         <Route exact path="/register" component={RegisterPage} />
         <Route exact path="/user" component={UserGreeting} />
         <Route exact path="/add-book" component={AddBookPage} />
+        <Route exact path="/book" component={BooksPage} />
```

Comment:
If the path `/book` is supposed to display a list of books then that should be reflected in its name. `books` or `book-list` are better alternatives